main.js

```diff
@@ -0,0 +1,34 @@
+const menuButton = document.querySelector('.mobile_menu');
+const mobileMenu = document.querySelector('#menu_top');
+const closeButton = document.querySelector('#close-button');
+
+function openMobileMenu() {
+  mobileMenu.style.display = 'block';
+}
+function closeMobileMenu() {
+  mobileMenu.style.display = 'none';
+}
+mobileMenu.style.display = 'none';
+menuButton.addEventListener('click', openMobileMenu);
+closeButton.addEventListener('click', closeMobileMenu);
+
+const menuItems = document.querySelectorAll('#menu_top a');
+
+menuItems.forEach((item) => {
+  item.addEventListener('click', function () {
+    closeMobileMenu();
+    const target = this.getAttribute('href');
+    document.querySelector(target).scrollIntoView({
+      behavior: 'smooth',
```

Comment:
- [OPTIONAL] I can see you tried to make the navigation to a different section of the webpage move on a smooth scroll, I noticed that this doesn't seem to work correctly. You can achieve your aim easily by adding this line of CSS at the top of your `style.css` file `html {
  scroll-behavior: smooth;
}` :wave: :waffle: